src/redux/checkout/tests/checkout.actions.test.js

```diff
@@ -1,18 +1,34 @@
 import {
   getNovaPoshtaCities,
-  setNovaPoshtaCities,
   getNovaPoshtaWarehouse,
+  getNovaPoshtaPrices,
+  setNovaPoshtaCities,
   setNovaPoshtaWarehouse,
   setNovaPoshtaPrices,
-  getNovaPoshtaPrices
+  getUkrPostRegions,
+  getUkrPostDistricts,
+  getUkrPostCities,
+  getUkrPostPostOffices,
+  setUkrPostRegions,
+  setUkrPostDistricts,
+  setUkrPostCities,
+  setUkrPostPostOffices
 } from '../checkout.actions';
 import {
-  SET_NOVAPOSHTA_CITIES,
   GET_NOVAPOSHTA_CITIES,
   GET_NOVAPOSHTA_WAREHOUSES,
+  GET_NOVAPOSHTA_PRICES,
+  SET_NOVAPOSHTA_CITIES,
   SET_NOVAPOSHTA_WAREHOUSES,
   SET_NOVAPOSHTA_PRICES,
-  GET_NOVAPOSHTA_PRICES
+  GET_UKRPOST_CITIES,
+  GET_UKRPOST_DISTRICTS,
+  GET_UKRPOST_POSTOFFICES,
+  GET_UKRPOST_REGIONS,
+  SET_UKRPOST_CITIES,
+  SET_UKRPOST_DISTRICTS,
+  SET_UKRPOST_POSTOFFICES,
+  SET_UKRPOST_REGIONS
 } from '../checkout.types';
 
 // cities actions tests
```

Comment:
please, remove this commented line, we can see such info in `describe`

Reply:
done